pages.es/common/mid3v2.md

```diff
@@ -0,0 +1,33 @@
+# mid3v2
+
+> Editar etiquetas de audio.
+> Ver también: `id3v2`.
+> Más información: <https://manned.org/mid3v2.1>.
+
+- Lista de todos los marcos ID3v2.3 o ID3v2.4 admitidos y sus significados:
+
+`id3v2 --list-frames {{ruta/a/archivo1.mp3 ruta/a/archivo2.mp3 ...}}`
```

Comment:
According to the [template](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md).
```suggestion
`id3v2 --list-frames {{ruta/al/archivo1.mp3 ruta/al/archivo2.mp3 ...}}`
```